internal/fs/inode/file.go

```diff
@@ -559,11 +559,9 @@ func (f *FileInode) Write(
 	data []byte,
 	offset int64) error {
 	// For empty GCS files also we will trigger bufferedWrites flow.
-	if f.src.Size == 0 && f.config.Write.EnableStreamingWrites {
```

Comment:
Changes in this file will go away once this PR is merged: https://github.com/GoogleCloudPlatform/gcsfuse/pull/2929